agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpClientDecorator.java

```diff
@@ -35,6 +36,10 @@
 
   protected abstract Integer status(RESPONSE response);
 
+  public Span getOrCreateSpan(REQUEST request, Tracer tracer) {
+    return getOrCreateSpan(spanNameForRequest(request), tracer);
+  }
+
   public String spanNameForRequest(final REQUEST request) {
```

Comment:
Can this be made private/protected now?

Reply:
Think it can when we migrate all the instrumentation to use `getOrCeateSpan`. Another PR?